tests/main/snap-core-fixup/task.yaml

```diff
@@ -25,7 +25,7 @@ execute: |
     umount /boot/uboot
 
     echo "Now test with the real corrupted image"
-    unxz -c -d test.img.xz >test.img
+    tar zxvf test.img.tar.gz
```

Comment:
What is the motivation behind this change? I would prefer to see this separately, with a comment explaining that, not bundled in a bigger "core18" change.

Reply:
@zyga this is because we don't have unxz on core18, so the alternative I found to make that test work on core18 is to use tar.